DQMOffline/L1Trigger/interface/L1TMuonDQMOffline.h

```diff
@@ -146,6 +149,7 @@ class L1TMuonDQMOffline : public DQMEDAnalyzer {
 
         // histos
         std::map<int, std::map<std::string, MonitorElement*> > m_EfficiencyHistos;
+        std::map<std::string, MonitorElement*> m_ResolutionHistos;        
```

Comment:
hi @astakia - please use a set of enums for the key of this new map to avoid numerous string comparisons below. Thanks.